src/map/MapRoutePath.js

```diff
@@ -4,7 +4,7 @@ import { useSelector } from 'react-redux';
 import { map } from './core/MapView';
 import { getSpeedColor } from '../common/util/colors';
 
-const MapRoutePath = ({ name, positions, coordinates }) => {
```

Comment:
Why are we removing `coordinates` here?

Reply:
it's not being used anymore

Reply:
@jcardus Looks like this is not true. You broke combined report. I appreciate contributions, but please make sure you do proper due diligence before sending pull requests.